stream-chat-android-ui-components/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="AvatarView">
+        <attr name="streamAvatarWidth" />
+        <attr name="streamAvatarHeight" />
+        <attr name="streamAvatarBorderWidth" />
+        <attr name="streamAvatarBorderColor" />
+        <attr name="streamAvatarTextSize" />
+        <attr name="streamAvatarTextColor" />
+        <attr name="streamAvatarTextFont" />
+        <attr name="streamAvatarTextFontAssets" />
+        <attr name="streamAvatarTextStyle" />
```

Comment:
Shouldn't all of these specify the `format` required?

Reply:
These attrs attrs are imported from `stream-chat-android` `attrs.xml`. Do you think it was better to duplicate them?

Reply:
Ah, I didn't realize they were already defined elsewhere. Looks all good then 👍